x-pack/plugins/lens/public/editor_frame_service/editor_frame/suggestion_panel.tsx

```diff
@@ -274,15 +274,14 @@ export function SuggestionPanel({
   contextRef.current = context;
 
   const AutoRefreshExpressionRenderer = useMemo(() => {
-    const autoRefreshFetch$ = plugins.data.query.timefilter.timefilter.getAutoRefreshFetch$();
     return (props: ReactExpressionRendererProps) => (
       <ExpressionRendererComponent
         {...props}
         searchContext={contextRef.current}
-        reload$={autoRefreshFetch$}
+        searchSessionId={frame.searchSessionId}
       />
     );
-  }, [plugins.data.query.timefilter.timefilter, ExpressionRendererComponent]);
+  }, [frame.searchSessionId, ExpressionRendererComponent]);
```

Comment:
This causes the component to get renewed each time the session id changes, which causes the loading bug again that got fixed with https://github.com/elastic/kibana/pull/86092

Please either put the search session id in a ref as well to avoid that or remove the `AutoRefreshExpressionRenderer` completely and just pass session id and search context down to the place where the component is actually used. With both of these approaches we can make sure the expression loader is not destroyed and re-created for each change.

I'm also seeing duplicated requests for the suggestions, but those might be caused by the same problem